Add user option for similarity measures (#40)
precice/micro-manager: docs/usage-configuration.md

Parameter | Description
--- | ---
`macro_domain_bounds`| Minimum and maximum bounds of the macro-domain, having the format `[xmin, xmax, ymin, ymax, zmin, zmax]` in 3D and `[xmin, xmax, ymin, ymax]` in 2D.

> *optional:* `adaptivity_similarity_measure`| Similarity measure to be used for adaptivity. Can be either `L1`, `L2`, `L1rel` or `L2rel`. By default, `L1` is used. The `rel` variants are the relative versions of the `L1` and `L2` norms, dividing the difference by the maximum value of the data.

Reviewer: ```suggestion
*optional:* `adaptivity_similarity_measure`| Similarity measure to be used for adaptivity. Can be either `L1`, `L2`, `L1rel` or `L2rel`. By default, `L1` is used. The `rel` variants calculate the respective relative norms.
```

Author: Do we not keep the explanation of the "relative-ness" in here?

Reviewer: > Do we not keep the explanation of the "relative-ness" in here?

That much detail is not necessary here. If someone really wants to know how we compute the relative norm, they are free to look into the code. What we do is per se not special.
